### ISSUE TITLE
Beförderungspapier: Anzeige der Informationen

### ISSUE THREAD

**Issue by StephanieFrancke** [SchweizerischeBundesbahnen/DAS#146]
# Ausgangslage
Als Lokpersonal muss ich vor Abfahrt im Ausgangs-, Unterwegs oder Wendebahnhof, im Besitz des Beförderungspapiers mit den gemäss RID vorgeschriebenen Angaben, des jeweils aktuellen ZIS Abschlusses (Status «kontrolliert»), sein. Zusätzlich muss ich in der Lage sein auch unterwegs jederzeit dieses Beförderungspapier RID einsehen zu können (offline Fähigkeit). 

Aktuell noch in Abklärung:
- Wird ZIS um folgende Anforderung erweitert?
Wenn Strassenfahrzeug mit gefährlichen Gütern auf Eisenbahnwagen befördert werden, muss anstelle der «Ladeeinheit Nummer» das Fahrzeugkennzeichen (Autonummer) des Strassenfahrzeuges auf dem Beförderungspapier RID abgebildet werden.
- Fahrzeugtyp: Wird der Fahrzeugtyp bei der Lok immer angegeben oder nur dann, wenn die EVN fehlt?
- Absender und Empfänger: Wird ZIS um die Adresse erweitert? Heute ist in ZIS nur die Abbildung einer Adresse eines Eisenbahn Verkehrsunternehmens (EVU) möglich, von Cargo ist die Adresse aber immer gewünscht.

# Weiterführende Links
- is blocked by: https://github.com/SchweizerischeBundesbahnen/DAS/issues/857
- Mögliche Darstellung Stand Mai 2025 - ZIS Abhängig: https://www.figma.com/design/b4EUSXE2AgrPu5Tv3HyEis/11--Stories-Formulierung?node-id=8683-438329&t=78g7MdbX9Odv8N2d-1

# Out of Scope
- #145 
- #603 
- #698 
- #697 
- #692 
- #540 
- #542 
- depends on: https://flow.sbb.ch/browse/ADFS-7679 (Umsetzung der aggregierten Informationen auf der ZIS-Schnittstelle und Trennung des zusätzlichen Informationen in einzelne Werte)
 
# Akzeptanzkriterien
- Das Beförderungspapier wird direkt aus dem Brems-/Lastzettel aufgerufen. Der Streckenabschnitt im angezeigten Beförderungspapier entspricht demjenigen vom Brems-/Lastzettel.
- Das Beförderungspapier (Wagenliste) kann mit folgenden übergreifenden Informationen aufgerufen werden:
  - Zuletzt geändert (wann wurde das Beförderungspapier angeliefert, Datum und Zeit)
  - Zugnummer (allenfalls mit Zusatz "R" für Rangierbewegungen auf der Strecke)
  - Datum
  - RRU (Responsible Railway Undertaking, ehemals SMS-EVU)
    - Da es keine ausführende EVU in ZIS gibt, wird diese EVU auch als "ausführende EVU" interpretiert.
- Ausserdem werden pro Wagen folgende Informationen angezeigt:
  - Abgangsbahnhof inkl. UIC Code
  - Position (inkl. Triebfahrzeug)
  - Wagennummer (EVN)
  - Fahrzeugtyp (ist nur vorhanden, wenn traktioniert)
  - Sendungsnummer (bestehend aus Versand EVU Company Code und Sendungsnummer)
  - Absender
  - Empfangsbahnhof inkl. UIC Code
  - Empfänger
  - Achsen
  - Länge über Puffer
  - Tara (Leergewicht Wagen)
  - Netto Gewicht
  - Brutto Gewicht
  - Vmax
  - Status der Handbremse
- Je Wagen kann es noch mehrere Einträge von Ladeeinheiten geben. Diese sollen mit folgenden Informationen angezeigt werden:
  - Ladungseinheitnummer
  - Sender
  - Empfänger 
- Das Beförderungspapier kann jederzeit wieder geschlossen werden, wodurch wieder der zugehörige Brems- und Lastzettel sichtbar wird.
# Testszenarien
## Vorbedingung:
- ...
## Durchführung:
- ...

# Team Review
- [ ] BA Review
- [ ] DEV Review
- [ ] PO Review

**Comment by StephanieFrancke** [comment]
@EverheardofUX : Die Darstellung vom Gefahrengut sollte mit Roland Meister und Dominic Gubser von I-SQU abgestimmt werden.

**Comment by EverheardofUX** [comment]
Noted, merci

Von: Stephanie Francke ***@***.***>
Datum: Montag, 21. Oktober 2024 um 15:11
An: SchweizerischeBundesbahnen/DAS ***@***.***>
Cc: Scanniello Desiree (IT-PTR-CEN1-SME2) ***@***.***>, Mention ***@***.***>
Betreff: Re: [SchweizerischeBundesbahnen/DAS] Beförderungspapier (Issue #146)

@EverheardofUX<https://github.com/EverheardofUX> : Die Darstellung vom Gefahrengut sollte mit Roland Meister und Dominic Gubser von I-SQU abgestimmt werden.

**Comment by mghilardelli** [comment]
> Wird ein neues Beförderungspapier für die geöffnete Fahrordnung geliefert während die Fahrordnung geöffnet ist, dann wird eine entsprechende Information angezeigt. -> nur mit ZIS anwendbar, muss bei Orca entfernt werden
@StephanieFrancke Sollen wir hier auch den Fall erwähnen, wenn eine Kontrolle der Formation zurückgezogen wird?
Zudem könnte man das noch genauer einschränken: wenn ein neues Beförderungspapier der geöffnete Fahrordnung und sich aktuell befindenden Teilstrecke geliefert wird... oder soll jede Änderung auf der ganzen Fahrordnung informiert werden?
Die erwähnten Punkte müssen auch bei #145 beachtet werden.

**Comment by mghilardelli** [comment]
> Das zugehörige Beförderungspapier wird beim Öffnen der Fahrordnung im Hintergrund geladen und steht auch offline zur Verfügung.

Offline ist nur gemeint, wenn es einmal geladen wurde (also die Fahrordnung geöffnet wurde) und nicht, dass es beim Vorladen (4h vorher) mit auf das Gerät synchronisiert wird?

**Comment by StephanieFrancke** [comment]
> > Wird ein neues Beförderungspapier für die geöffnete Fahrordnung geliefert während die Fahrordnung geöffnet ist, dann wird eine entsprechende Information angezeigt. -> nur mit ZIS anwendbar, muss bei Orca entfernt werden
> 
> @StephanieFrancke Sollen wir hier auch den Fall erwähnen, wenn eine Kontrolle der Formation zurückgezogen wird? Zudem könnte man das noch genauer einschränken: wenn ein neues Beförderungspapier der geöffnete Fahrordnung und sich aktuell befindenden Teilstrecke geliefert wird... oder soll jede Änderung auf der ganzen Fahrordnung informiert werden? Die erwähnten Punkte müssen auch bei #145 beachtet werden.

Ja, ich würde jegliche Änderung kennzeichnen, da wir nicht wissen, was das Lokpersonal eventuell bereits im Vorfeld angeschaut hat.

**Comment by StephanieFrancke** [comment]
> > Das zugehörige Beförderungspapier wird beim Öffnen der Fahrordnung im Hintergrund geladen und steht auch offline zur Verfügung.
> 
> @StephanieFrancke Offline ist nur gemeint, wenn es einmal geladen wurde (also die Fahrordnung geöffnet wurde) und nicht, dass es beim Vorladen (4h vorher) mit auf das Gerät synchronisiert wird?

Ja genau, nur beim Öffnen der entsprechenden Fahrordnung.
